code/_onclick/click.dm

```diff
@@ -446,7 +446,10 @@
 
 /// MouseWheelOn
 /mob/proc/MouseWheelOn(atom/A, delta_x, delta_y, params)
-	return
+	if(incapacitated(ignore_restraints = TRUE, ignore_stasis = TRUE))
+		return
+
+	SEND_SIGNAL(src, COMSIG_MOUSE_SCROLL, delta_x, delta_y, params)
```

Comment:
```suggestion
	SEND_SIGNAL(src, COMSIG_MOUSE_SCROLL, A, delta_x, delta_y, params)

```
As before, you need to pass all parameters, because this is a generic signal.
The incapacitated check is irrelevant to the event, that needs to be done on the proc that reacts to it, instead.